docker/etc/settings.py

```diff
@@ -21,8 +21,8 @@
 on the per environment basis. e.g: export PULP_GALAXY_....
 """
 
-X_PULP_CONTENT_HOST = "pulp-content-app"
-X_PULP_CONTENT_PORT = 24816
+X_PULP_CONTENT_HOST = os.environ.get('X_PULP_CONTENT_HOST', 'pulp-content-app')
+X_PULP_CONTENT_PORT = os.environ.get('X_PULP_CONTENT_HOST', 24816)
```

Comment:
I am ok with this, but looks like the name must be full, including the prefix, this is using `os` to access env directly and os does not strip out the prefix

```suggestion
X_PULP_CONTENT_HOST = os.environ.get('PULP_X_PULP_CONTENT_HOST', 'pulp-content-app')
X_PULP_CONTENT_PORT = os.environ.get('PULP_X_PULP_CONTENT_HOST', 24816)
```

Reply:
fixed, thank you :+1: